app/views/examples/button-groups/index.njk

```diff
@@ -0,0 +1,111 @@
+{% extends "layout.njk" %}
```

Comment:
We could also update could `/full-page-examples/confirm-delete` to use the new button group.

Reply:
Ah great call, thanks :+1:

Reply:
Done in f85eb47